meta-resin-common/recipes-support/bindmount/bindmount_0.0.1.bb

```diff
@@ -0,0 +1,36 @@
+# Auto-Generated by cargo-bitbake 0.3.9
+#
+inherit cargo
+
+# If this is git based prefer versioned ones if they exist
+# DEFAULT_PREFERENCE = "-1"
+
+# how to get bindmount could be as easy as but default to a git checkout:
+# SRC_URI += "crate://crates.io/bindmount/0.0.1"
+SRC_URI += "git://git@github.com/resin-os/resinos-bindmount.git;protocol=ssh"
+SRCREV = "f597fb026636f0f1be1fbfe2f5a8c2edee3c4493"
+S = "${WORKDIR}/git"
+CARGO_SRC_DIR=""
+PV_append = ".AUTOINC+f597fb0266"
+
+# please note if you have entries that do not begin with crate://
+# you must change them to how that package can be fetched
+SRC_URI += " \
+crate://crates.io/errno/0.2.3 \
+crate://crates.io/kernel32-sys/0.2.2 \
+crate://crates.io/libc/0.2.36 \
+crate://crates.io/winapi-build/0.1.1 \
+crate://crates.io/winapi/0.2.8 \
```

Comment:
missing tab or 4 spaces here

Reply:
This is an autogenerated recipe from `cargo bitbake`.